webpack.config.js

```diff
@@ -17,8 +21,8 @@ module.exports = {
 	},
 	devtool: "source-map",
 	output: {
-		path: path.resolve(__dirname, "client/js"),
-		filename: "bundle.js",
+		path: path.resolve(__dirname, "client"),
+		filename: "[name]",
```

Comment:
Usually with webpack, you'll be using the compilation hash that can easily be appended at build.

You would configure webpack to output like so inside the webpack config...

```
module.exports = {
...
	output: {
		path: path.join(__dirname, 'dist'),
		pathinfo: (env !== 'production'),
		filename: 'app-[hash:6].js'
       }
...
}
```

This will output a app-rand6c.js (app- 6 random characters as i've requested 6). filename in the output folder.

From there you could include it easily in the server.js